frappe/__init__.py

```diff
@@ -248,6 +249,7 @@ def init(site: str, sites_path: str = ".", new_site: bool = False, force=False)
 	local.dev_server = _dev_server
 	local.qb = get_query_builder(local.conf.db_type or "mariadb")
 	local.qb.engine = get_qb_engine()
```

Comment:
need to break this. the refactored `Engine` doesn't clear any properties from the previous query

Reply:
Ah didn't notice this :+1:

Reply:
we could do a soft break though: qb.engine = frappe._dict(get_query=qb.get_query)